lib/ansible/modules/files/file.py

```diff
@@ -79,12 +79,13 @@
 '''
 
 EXAMPLES = '''
-# change file ownership, group and mode. When specifying mode using octal numbers, first digit should always be 0.
+# change file ownership, group and mode
 - file:
     path: /etc/foo.conf
     owner: foo
     group: foo
     mode: 0644
+# when specifying mode using octal numbers, add a leading 0
 - file:
```

Comment:
comments should go above the thing they're commenting.  you can put it above the mode line, though